### DISCUSSION TITLE
Audio too quiet

### DISCUSSION THREAD

**Discussion by RK11111111111** [fayaaz/mixxx-pi-gen#77] [General]
I've noticed that the Audio is very quiet.  I think it has something to do with the configuration.  I have tested my skins and  controller settings with my laptop and the audio seems fine with these configuration files.  I also used to use the controller with an iPad and mix vibes cross DJ, the audio was a little quiet, but still not this bad.  I think it is due to the iPad sound leveling Luffs.

In Mixxx I disabled the ReplayGain Normalization, which helped.  I also tried to crank up the LUFS with the replay normalization which worked, but produced a very distorted signal, an issue I hadn't had before at that volume level.

So, my inclination is that it is something with the audio settings in or mixxx.cfg.  I tried to change ALSA settings with the mixer software, but it didn't seem to recognize any audio interface.

Right now it is still useable, but I have to crank the amplifier and turn everything else down at the mixer, introducing a bit of noise.

Any ideas on where to start trouble shooting this.  I tried the ALSA settings, but it didn't appear to recognize the audio, maybe this is the problem?

**Comment by fayaaz** [comment]
Could you try running `alsamixer` on your terminal and checking the volumes? 
Some devices start on a low volume (I had this problem with a Behringer UMC204HD on recent Linux kernel versions)

**Reply by RK11111111111** [reply]
I think this was the problem.  The interface was set at -20db gain.  I had tried running it from an ssh terminal, but alsamixer needs an interactive terminal.
